### PR TITLE
Link Rainbow paper in training scripts

### PR DESCRIPTION
**Pull request by marioyc**
Some scripts were referencing the C51 paper while using Rainbow's epsilon for Adam

### CODE REVIEW

File: examples/slimevolley/train_rainbow.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def phi(x):
     # Turn off explorer
     explorer = explorers.Greedy()
 
-    # Use the same hyper parameters as https://arxiv.org/abs/1707.06887
+    # Use the same eps as https://arxiv.org/abs/1710.02298
```
Comment (prabhatnagarajan) [inline examples/slimevolley/train_rainbow.py:179]: Shouldn't we remove this comment? It doesn't use the same learning rate as `train_rainbow.py`:

image

Comment (marioyc) [reply examples/slimevolley/train_rainbow.py:179]: I kept it since we're still taking the eps value from there, I guess that was the original idea

Comment (prabhatnagarajan) [reply examples/slimevolley/train_rainbow.py:179]: Hmm. Sure.
